### PR TITLE
Sac user auth new

### PR DESCRIPTION
**Pull request by imsks**
1. Created a new branch as sac-user-auth-new
2. rebased with master and sac-user-auth

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Environment variables declared in this file are automatically made available to Prisma.
```
Comment (SanjayBisht96) [inline .env]: .env file should me committed in git. Keep it in local.

File: api-utils/auth.js [hunk 2]
```diff
@@ -1,6 +1,7 @@
 const jwt = require("jsonwebtoken");
+const bcrypt = require("bcrypt");
 
-exports.getAuthTokens = (payload) => {
+export const getAuthTokens = (payload) => {
   // Generate Access Token
   let accessToken = jwt.sign(payload, process.env.ACCESS_TOKEN_SECRET, {
     algorithm: "HS256",
```
Comment (SanjayBisht96) [inline api-utils/auth.js:7]: create a consts file to save magic numbers and import constant values from their. You can add consts.js file in root folder

File: next.config.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
+module.exports = {
```
Comment (SanjayBisht96) [inline next.config.js:1]: why using this file? Though env file is there?

File: sample.env [hunk 4]
```diff
@@ -0,0 +1,12 @@
+ENV=0
```
Comment (SanjayBisht96) [inline sample.env:1]: Don't commit this file

File: styles/pages/_homepage.scss [hunk 5]
```diff
@@ -1,47 +0,0 @@
-.ipaddressapp {
```
Comment (SanjayBisht96) [inline styles/pages/_homepage.scss:1]: try using tailwind for css

File: next.config.js (hunk 3 above)
Comment (imsks) [reply next.config.js:1]: I wasn't taking anything from env files that's why

File: sample.env (hunk 4 above)
Comment (imsks) [reply sample.env:1]: It's just a sample file. Khaja asked me to commit it

File: styles/pages/_homepage.scss (hunk 5 above)
Comment (imsks) [reply styles/pages/_homepage.scss:1]: I can very fluent with SCSS so can we use both.

File: .env (hunk 1 above)
Comment (imsks) [reply .env]: it's already committed. Maybe came with master branch

File: api-utils/auth.js (hunk 2 above)
Comment (imsks) [reply api-utils/auth.js:7]: I will add the file

**Pull request merged**